pkg/importer/avro/transformer.arrai

```diff
@@ -0,0 +1,105 @@
+let util = //{./avro_util};
+
+# Extra items transformed to Sysl type !enum
```

Comment:
I'm confused as to why these are "extra". What does "extra" mean in this context?